packages/math-input/src/components/keypad/__tests__/keypad-v2-mathquill.test.tsx

```diff
@@ -68,7 +68,7 @@ function V2KeypadWithMathquill(props: Props) {
         <div style={{maxWidth: "400px", margin: "2em"}}>
             <Popover
                 content={
-                    <div>
+                    <PopoverContentCore>
```

Comment:
The Wonder Blocks `Popover` tries to restrict the `content` prop to either a `PopoverContent` or `PopoverContentCore` element, but Typescript doesn't properly recognize that arbitrary JSX isn't valid.